### PR TITLE
chore(likes): prevent PHP notice for missing key in $vars

### LINKED ISSUE #9994: Undefined index: list_class in "mod\likes\views\default\likes\before_lists.php" (line 12)
if `$vars['list_class']` is not set (for whatever reason) a php notice is issued

### PR DESCRIPTION
**Pull request by jeabakker**
fixes #9994

### CODE REVIEW

File: mod/likes/views/default/likes/before_lists.php [hunk 1]
```diff
@@ -7,9 +7,10 @@
  */
 
 $user = elgg_get_logged_in_user_entity();
+$list_class = elgg_extract('list_class', $vars);
```
Comment (hypeJunction) [inline mod/likes/views/default/likes/before_lists.php]: IMO, `explode(' ', $list_class)` and then check if it's in array?

Comment (jeabakker) [inline mod/likes/views/default/likes/before_lists.php]: changed

**Review by hypeJunction — APPROVED** [review]

**Comment by jeabakker** [comment]
@mrclay did a fix for 2.2 in #10164

**Comment by mrclay** [comment]
If you fix a different way, please take care to merge 1.12 into 2.2 so you can handle the conflict.

**Comment by jeabakker** [comment]
I can live with the fact that you fixed it and close this PR

**Pull request closed without merging**